docs/Using Price Feeds/get-the-latest-price-solana.md

```diff
@@ -16,24 +16,24 @@ Price data feeds are available on the following networks:
 - **Solana network**
 - [EVM-compatible networks](../get-the-latest-price/)
 
-The full list of price data feeds for each network is available on the [Price Feed Contracts](../reference-contracts/) page.
+The full list of price data feeds for each network is available on the [Data Feed Contracts](../reference-contracts/) page.
 
-Chainlink Price Feeds are live on the [Solana Devnet](https://explorer.solana.com/?cluster=devnet) with sub-second updates. Chainlink’s Solana deployment has no dependencies on external blockchain networks such as Ethereum. To learn more about the Solana programming model, see the [Solana Documentation](https://docs.solana.com/developing/programming-model/overview).
+Chainlink Data Feeds are live on the [Solana Devnet](https://explorer.solana.com/?cluster=devnet) with sub-second updates. Chainlink’s Solana deployment has no dependencies on external blockchain networks such as Ethereum. To learn more about the Solana programming model, see the [Solana Documentation](https://docs.solana.com/developing/programming-model/overview).
 
-To get the full list of Chainlink Price Feeds running on the Solana Devnet, see the [Solana Feeds](/docs/solana-price-feeds/) page.
+To get the full list of Chainlink Data Feeds running on the Solana Devnet, see the [Solana Feeds](/docs/solana-price-feeds/) page.
 
 You can view the program ID that owns these feeds in the [Solana Devnet Explorer](https://explorer.solana.com/address/2yqG9bzKHD59MxD9q7ExLvnDhNycB3wkvKXFQSpBoiaE?cluster=devnet).
 
 > 📘 Note
 >
-> This guide shows you how to deploy and retrieve price data using the Solana network. If you need to use EVM-compatible networks, see [Consuming Price Feeds (EVM)](../get-the-latest-price/).
+> This guide shows you how to deploy and retrieve data using the Solana network. If you need to use EVM-compatible networks, see [Consuming Data Feeds (EVM)](../get-the-latest-price/).
```

Comment:
Should we fix this URL and forward?

Reply:
We will handle URL changes and redirects in a separate change. There are several.